package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web-fonts",
-  "version": "1.0.2",
+  "version": "1.0.4",
```

Comment:
Because I suck at Semver and forgot to increment the package when I released 1.0.3